library/system/io.lean

```diff
@@ -3,17 +3,17 @@ Copyright (c) 2016 Microsoft Corporation. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Luke Nelson, Jared Roesch and Leonardo de Moura
 -/
-constant {u} io : Type u → Type u
-constant io.functor : functor io
-constant io.monad   : monad io
+meta constant {u} io : Type u → Type u
+meta constant io.functor : functor io
```

Comment:
I see.  Then we should probably only print the axioms (transitively) referenced by the specified theorems.